tests/queries/0_stateless/02310_clickhouse_local_INSERT_progress_profile_events.expect

```diff
@@ -27,12 +24,27 @@ expect_after {
     -i $any_spawn_id timeout { exit 1 }
 }
 
-spawn bash
-send "source $basedir/../shell_config.sh\r"
+# We use min_chunk_bytes_for_parallel_parsing and input_format_parallel_parsing
+# to form tiny blocks and make progress output more interactive.
+#
+# Using standard chunk sizes would require pushing much larger data amounts, which
+# would be too slow on sanitizer and debug builds, potentially causing timeouts.
+#
+# interactive_delay allows progress to be printed more frequently.
+# The sleep command in the bash loop ensures that interactive_delay will be reached.
+spawn bash -c "
+    source $basedir/../shell_config.sh; \
+    for i in {1..10}; do \
+        yes | head -n 10; \
```

Comment:
Hm, I guess that 10 rows in a chunk is too small, initially (#37391) I used 10e6, then decreased to 1e6, but 10 is just too low, but in this case it will be better to remove the test completely, since now it is already hard to ensure that it verifies something.

Reply:
Why exactly you need so many rows? I thought the test was about checking that the progress works for the inserts with  clickhouse-local? 
WIth those (tuned) settings you can get progress prints frequently.

Reply:
The test is for this line - https://github.com/ClickHouse/ClickHouse/pull/37391/files#diff-22cea532543e7c87463eeb73d6815661d0c3f9e621ea915c99a9a9334ac4008aR57

But after thinking about this more I think your changes is OK, and indeed correct, big block is not required only number of progress packets is important.

Reply:
But let's increase number of packets, 10*10 is not big enough, let's make it i.e. 10000